common/decklist.cpp

```diff
@@ -579,8 +579,6 @@ bool DeckList::loadFromStream_Plain(QTextStream &in)
         cardName.replace(rx, "'");
         rx.setPattern("Æ");
         cardName.replace(rx, "AE");
```

Comment:
I thought there are supposed to be no more `Æ` cards at all starting with Kaladesh?
(besides the ordinary errors on gatherer for sure...)